exp-android-sdk/src/main/java/com/scala/exp/android/sdk/SocketManager.java

```diff
@@ -3,6 +3,7 @@
 
 import android.util.Log;
```

Comment:
I think handleBroadCast and handleRequest need to be updated to use the channelCache right?  right now they just use the enums to route messages. Also handleBroadCast() should have the name fixed: handleBroadcast()